config.properties.example

```diff
@@ -18,7 +18,7 @@ recaptchaPublicKey=RECAPTCHA_PUBLIC_KEY
 recaptchaPrivateKey=RECAPTCHA_PRIVATE_KEY
 
 # Optional: Default value set to http://accounts.jenkins.io/
-url=http://localhost:8080
+url=APP_URL
```

Comment:
=/ great a file called example actually matters in production... iirc I had issues locally with it set to this but i'll look later

Reply:
yep, sorry I went for a quick rollback but we can definitvely improve this (like having the java app reading en vars instead of having an entrypoint.sh in the docker imag which "seds" the env vars 🥲

Reply:
right choice in the short term, for later: https://github.com/jenkins-infra/account-app/issues/162